components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/logout/LogoutRequestSender.java

```diff
@@ -173,47 +179,45 @@ public void run() {
                 log.debug("SAMLRequest : " + decodedSAMLRequest);
             }
 
+            String hostNameVerificationEnabledProperty =
+                    IdentityUtil.getProperty(IdentityConstants.ServerConfig.SLO_HOST_NAME_VERIFICATION_ENABLED);
+            boolean isHostNameVerificationEnabled = StringUtils.isNotBlank(hostNameVerificationEnabledProperty) ?
```

Comment:
Here, the default behaviour should be 'true' unless it's explicitly set to 'false'.

So I usually handle this in below manner,

```
boolean isHostNameVerificationEnabled = true;
if ( "false".equalsIgnoreCase(isHostNameVerificationEnabled)) {
    isHostNameVerificationEnabled = false;
}
```
So it will handle the config in mentioned manner.